lib/bridge/MatrixHandler.js

```diff
@@ -1486,6 +1517,89 @@ MatrixHandler.prototype._onUserQuery = Promise.coroutine(function*(req, userId)
     yield this.ircBridge.getMatrixUser(ircUser);
 });
 
+MatrixHandler.prototype._textForReplyEvent = Promise.coroutine(function*(event, ircRoom) {
+    const REPLY_REGEX = /> <(@.*:.*)>(.*)\n\n(.*)/;
```

Comment:
I'm not sure if we actually need to perform this regex at all - the existence of `m.relates_to/m.in_reply_to` should be enough for us to consider it a reply, regardless of how silly the sender's client is.

Reply:
nevermind, we need the actual text the user sent... I wonder if it's worth using the html instead as one common thing amoung reply implementers is mixing up how the fallback actually works.